shared/src/simulation/events/transfer-vehicles-request.ts

```diff
@@ -41,12 +44,14 @@ export class TransferVehiclesRequestEvent implements SimulationEvent {
      */
     constructor(
         requestedVehicles: ResourceDescription,
+        transferInitiatingRegionId: UUID | undefined,
```

Comment:
```suggestion
        transferInitiatingRegionId?: UUID,
```

For consistency with the class member.